python/whylogs/experimental/core/metrics/udf_metric.py

```diff
@@ -345,7 +346,7 @@ def upper(x):
     """
 
     resolvers = generate_udf_resolvers()
-    non_udf_resolvers = non_udf_resolvers if non_udf_resolvers is not None else STANDARD_RESOLVER
+    non_udf_resolvers = non_udf_resolvers if non_udf_resolvers is not None else STANDARD_PRIVACY_PRESERVING_RESOLVER
```

Comment:
see line 64

Reply:
Can you elaborate?

Reply:
It's defaulting to `STANDARD_RESOLVER` for `UdfMetric` submetrics.

Reply:
Right, which I think is what we want. We don't want the full metrics on the raw inputs, so that the focus is on the UDF output which should get the standard resolver.

Reply:
That will still include frequent items on the UDF outputs.

Reply:
its a bit heavy handed here, basically we are lowering the number of metrics you get on columns even when the UDFs aren't targeting them if you have a mix of UDFs and non-UDF columns. I'm not suggesting this goes straight to mainline, but I'd like to test with this in a dev build since the API is closer to the simple calling pattern I think we want to get to for UDF cases.

Reply:
We don't have to merge this at all, but I think I'll create a dev build here to get some feedback on the impact to callers integrating with UDFs. Maybe a thumbs up here would do for me, mostly just looking for imput from @richard-rogers before I create the next dev build